MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/MuonFitPropertyBrowser.h

```diff
@@ -149,6 +190,32 @@ private slots:
   std::string m_simultaneousLabel;
   QtProperty *m_normalization;
   mutable QStringList m_normalizationValue;
```

Comment:
these are all overwritten at other points in the code. 

m_normalization -> in TFAsymmfit
m_groupsToFitOptions -> in addGroupCheckbox
m_ShowGroupValues -> in updateGroupDisplay

similarly for periods.